src/services/comment.service.ts

```diff
@@ -1,77 +1,130 @@
-
 import daoset from '../models';
+import jwt from "jsonwebtoken";
 const cmtDao = daoset.cmtDao;
 const userDao = daoset.userDao;
```

Comment:
```suggestion
import jwt from "jsonwebtoken";
import { cmtDao, userDao } from "../models";
```
import 병합 가능할 것 같아요!

Reply:
반영했습니다. 감사합니다~!